nodeoperators/nodelist.json

```diff
@@ -0,0 +1,2826 @@
+[
+    {
+        "Role": "verification",
+        "Address": "verification-009.mainnet4.nodes.onflow.org:3569",
```

Comment:
shouldn't these be `mainnet5` nodes

Reply:
they're stuck as mainnet4 in the smart contract, but i guess i can update them here